oeps/best-practices/oep-0066-bp-authorization.rst

```diff
@@ -0,0 +1,395 @@
+OEP-66: User Authorization
+###########################
+
+.. list-table::
+   :widths: 25 75
+
+   * - OEP
+     - :doc:`OEP-66 <oep-0066-bp-authorization>`
+   * - Title
+     - User Authorization
+   * - Last Modified
+     - 2023-09-12
+   * - Authors
+     - Hilary Sinkoff (hsinkoff@2u.com), Jeremy Bowman (jbowman@edx.org)   
+   * - Arbiter
+     - Feanil Patel
+   * - Status
+     - Draft
+   * - Type
+     - Best Practice
+   * - Created
+     - 2023-08-21
+   * - `Review Period`
+     - 2023-09-12 - 
+   * - Replaces
+     - :doc:`oep-0009-bp-permissions`
```

Comment:
I was about to ask how this OEP was related to the OEP-9. Now, should we use the `Motivation` section from the OEP-9 or write a new one explaining the replacement?

Reply:
Related, you might want to change the Status of OEP-9 to "Superseded" in this PR, if that is your intention.

Reply:
Marking resolved.